services/repository-managers/src/main/java/org/sagebionetworks/repo/manager/oauth/OIDCTokenHelperImpl.java

```diff
@@ -150,7 +150,7 @@ public String createAnonymousAccessToken() {
 		String subject = AuthorizationConstants.BOOTSTRAP_PRINCIPAL.ANONYMOUS_USER.getPrincipalId().toString(); // we don't encrypt the subject
 		String oauthClientId = ""+AuthorizationConstants.SYNAPSE_OAUTH_CLIENT_ID;
 		String tokenId = UUID.randomUUID().toString();
-		List<OAuthScope> noScopes = Collections.EMPTY_LIST;
+		List<OAuthScope> noScopes = Collections.singletonList(OAuthScope.view);
```

Comment:
This should be full scope as the ACL will take care of access restrictions

Reply:
Done.